spec/javascript/utils/SetupBrowserAndPage.js

```diff
@@ -15,7 +15,7 @@ const IgnoreImageAndCSSLoad = async (page) => {
 }
 
 const SetupBrowser = async () => {
-  let browser = await puppeteer.launch({ headless })
```

Comment:
this looks not right. headless and removecss should be separate things and we should still be indicating whether or not it's headless?

Reply:
this is still an issue. we need to be able to run non-headless locally. I'll fix it